composer.json

```diff
@@ -25,11 +25,11 @@
         "sort-packages": true
     },
     "require": {
-        "php": ">=5.5.0"
+        "php": ">=5.6.0"
     },
     "require-dev": {
         "phpmd/phpmd": "2.*",
```

Comment:
Not sure if we need the other dependencies at all. When I take a look at the commit hook it feels like they have not been executed since some time...